apps/artwork_purchase/index.styl

```diff
@@ -86,7 +87,7 @@ a#purchase-page-header-title
       // margin 30px
       garamond-size('l-body')
       clearfix()
-      &:not(.ap-summary-price) .ap-summary__right
+      &:not(.ap-summary__price) .ap-summary__right
```

Comment:
Wait maybe I'm wrong. I was trying to figure out where the color of the price is changed though?